CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+## [1.21.3] - 2021-08-13
```

Comment:
let's do a minor version bump, since a new API in the channel options (`SkipHandlerMethods`)